checks/security_policy.go

```diff
@@ -34,6 +34,9 @@ func SecurityPolicy(c *checker.CheckRequest) checker.CheckResult {
 		if strings.EqualFold(name, "security.md") {
 			logf("security policy : %s", name)
 			return true, nil
+		} else if strings.EqualFold(name, "**/security.rst") {
```

Comment:
we should not allow arbitrary paths here since security policy should be discoverable. i think that helper function i am saying can account for docs/ directory in root, but lets not allow any other paths ?

Reply:
Took care of it! Now it will just look at either `doc` or `docs` folder